models/engine/file_storage.py

```diff
@@ -68,3 +68,16 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """ Retrieves an object given its class name and id """
+        if cls and id:
+            objs = self.all(cls)
+            key = cls.__name__ + '.' + id
+            return objs.get(key)
+        else:
+            return None
+
+    def count(self, cls=None):
+        
```

Comment:
Nice code Diego but you forget to  comment your function.

Reply:
ALright, I'll check now!